AdobeStockAssetApi/Api/AssetRepositoryInterface.php

```diff
@@ -40,7 +40,9 @@ public function delete(AssetInterface $item): void;
      * Get a list of assets
      *
      * @param \Magento\Framework\Api\SearchCriteriaInterface $searchCriteria
+     *
      * @return \Magento\AdobeStockAssetApi\Api\Data\AssetSearchResultsInterface
+     * @throws \Magento\Framework\Exception\IntegrationException
      */
     public function getList(SearchCriteriaInterface $searchCriteria) : AssetSearchResultsInterface;
```

Comment:
Retrieving assets should be performed from the database and should not result in calls to Adobe Stock API, so there should not be a chance of `\Magento\Framework\Exception\IntegrationException` being thrown